src/Helper/Mailer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Helper;
+
+use Yiisoft\Mailer\MailerInterface;
+
+final class Mailer
```

Comment:
That's not a helper per se. I'd name it a service and put it along with `Message` into `App\Mailer`.

Reply:
done.